lib/cli.mjs

```diff
@@ -383,6 +383,8 @@ const meowOptions = {
  * @returns {Promise<any>}
  */
 export default async function main(argv) {
+	process.exitCode = 0;
```

Comment:
question

Isn't `0` the default?
If so, shouldn't the reset be handled in the tests themselves?

Reply:
@Mouvedia I think not setting `process.exitCode` is effectively the same as setting it to 0. Yeah, I agree resetting `process.exitCode` in tests is better. Let me double check with @ybiquitous on this because I am a bit confused of his comment.

@ybiquitous Could you clarify a bit more the reason of adding `process.exitCode = undefined;` in `afterAll()`? Is the purpose to make sure the final exit code of `npm run test` not affected by our test cases? Or do you mean resetting `process.exitCode` in `afterEach()` rather than `afterAll()`?

I agree with @Mouvedia about resetting `process.exitCode` directly in tests rather than in `cli.mjs`. Here is my plan if that sounds good to you:
- Remove `process.exitCode = EXIT_CODE_SUCCESS;` from the `main` function.
- Add `afterEach(() => { process.exitCode = undefined; });` in `cli.test.mjs`.
- Replace `expect(process.exitCode).toBe(0);` with `expect(process.exitCode).toBeUndefined();` in `cli.test.mjs`.

@ybiquitous what do you think?

Reply:
👍 for `afterEach`

Reply:
Ah, sorry. `afterEach` is better 👍🏼

Reply:
Thanks! I have updated the CLI code and the corresponding tests in fa98abe77234da7e949c2351cb80611af4c8a408.